cli/src/main/java/com/devonfw/tools/ide/tool/LocalToolCommandlet.java

```diff
@@ -394,19 +394,20 @@ private void installDependencies(DependencyInfo dependencyInfo, EnvironmentConte
     VersionIdentifier dependencyVersionToInstall = this.dependency.findDependencyVersionToInstall(dependencyInfo);
 
     Path dependencyRepository = getDependencySoftwareRepository(dependencyName, dependencyTool.getConfiguredEdition());
+    String pathVariable = this.tool.toUpperCase(Locale.ROOT) + "_HOME";
```

Comment:
As stated above this is incorrect. Assuming here you are in the `Tomcat` commandlet and would decide that the variable needs to be set to `JAVA_HOME`. Maybe the `dependencyTool` which is `Java` in this example needs to decide something else instead of this default, this cannot be solved properly.
E.g. Maven might need `MAVEN_HOME` instead of `MVN_HOME` just to give another example.

Reply:
I have reverted this change 👍